onboard/catkin_ws/src/cv/scripts/test_images.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import rospy
+import cv2
+import os
+from custom_msgs.srv import ToggleModel
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+
+
+# Mock the camera by publishing the same image to a topic
+class DummyImagePublisher:
+
+    NODE_NAME = 'test_images'
+    CAMERA = 'left'
+    IMAGE_TOPIC = '/camera/{}'.format(CAMERA)
+    # IMAGE_TOPIC = '/test_images/image'
+
+    # Read in the dummy image and other misc. setup work
+    def __init__(self):
+        self.image_publisher = rospy.Publisher(self.IMAGE_TOPIC, Image, queue_size=10)
+
+        path = os.path.dirname(__file__)
+        image = cv2.imread(os.path.join(path, '../assets/buoy.jpg'), cv2.IMREAD_COLOR)
```

Comment:
Hardcoded file? Is this just a test script?

Reply:
Yeah it's a test script

Reply:
Correct -- this is just a test script; we won't actually be using this. Essentially we're publishing a dummy video feed using the buoy picture in /assets/buoy.jpg